apps/backend/src/publishHandler.ts

```diff
@@ -278,6 +284,7 @@ export const publishHandler: RouteHandler<{
           engine: codemodRc.engine,
           applicability: codemodRc.applicability,
           verified: isVerified,
+          hidden: isPublishedFromStudio,
```

Comment:
What is difference between `hidden` and `private`?

Reply:
hidden is meant to hide the codemod from being visible in the registry/CLI by default. the codemods that are published from studio by using `Run via CLI` feature are considered as temporary versions of the codemods that are not meant to be seen implicitly. so we hide them by default and give user an option to view them (registry switch is in progress)

Reply:
even if the codemod is private, it will show up in the registry/CLI for the user.